_guide_topics/rhinopython/python-procedures.md

```diff
@@ -90,13 +90,15 @@ The end of the function is marked by the loss of whitespace in the next line of
 In the following example, the Celsius `def` calculates degrees Celsius from degrees Fahrenheit. When the def is called from the `ConvertTemp` def procedure, a variable containing the argument value is passed to the def.  The result of the calculation is returned to the calling procedure and displayed in a message box.
 
 ```python
+import rhinoscriptsyntax as rs
+
 def Celsius(fDegrees):
-    _Celsius = (fDegrees - 32) * 5 / 9
+    _Celsius = (float(fDegrees) - 32) * 5 / 9
     return _Celsius;
 
 # Use this code to call the Celsius function
-temp = raw_input("Please enter the temperature in degrees F.", 1)
-MsgBox "The temperature is " & Celsius(temp) & " degrees C."
+temp = raw_input("Please enter the temperature in degrees F.")
+rs.MessageBox("The temperature is " + str(Celsius(temp)) + " degrees C.")
```

Comment:
MessageBox is a component of rhinoscriptsyntax and should look like this change.